js/main.js

```diff
@@ -25,5 +25,81 @@ $form.addEventListener('submit', function (event) {
 
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
 
+  $entryformDiv.className = 'hidden';
+  $entriesDiv.className = 'entries';
+  data.view = 'entries';
   $form.reset();
+  window.location.reload();
+});
+
+function treeMaker(entry) {
+
+  var ulElement = document.createElement('ul');
```

Comment:
You shouldn't make the `ul` here, it should be in the HTML document. You want this to just create the `li` and return the `li`. That way you can use this function in your `submit` event handler as well to make a single entry.